include/powsybl/iidm/VoltageLevel.hpp

```diff
@@ -41,7 +40,7 @@ class Terminal;
 class VscConverterStation;
 class VscConverterStationAdder;
 
-class VoltageLevel : public Container, public MultiVariantObject {
+class VoltageLevel : public Container {
```

Comment:
To be checked: does the implementation of MultiVariantObject methods ok (super call?)

Reply:
VoltageLevel does not have any custom implementation of MultiVariantObject methods.